docs/docs/css-libraries-and-frameworks.md

```diff
@@ -5,6 +5,6 @@ overview: true
 
 There are many other CSS libraries and frameworks that you can use in your Gatsby project.
 
-These are not full-on approaches to styling, and generally work no matter which styling approach you've chosen for your website. They require installing third-party libraries, often with the help of Gatsby community plugins.
+These style libraries are not full-on approaches to styling, and generally work no matter which styling approach you've chosen for your website. They require installing third-party libraries, often with the help of Gatsby community plugins.
```

Comment:
This sentence ended up pretty repetitive, as it includes the word "styling" 3 times. I'd recommend putting it in the first sentence instead ("There are many other CSS styling libraries and frameworks...")

Reply:
Awesome!! Thank you :)